### ISSUE TITLE
libgconf-2-4 is not available in Ubuntu 23.10

### ISSUE THREAD

**Issue by josteinaj** [keybase/client#25984]
Running:
```sh
LANG=en sudo apt install ./keybase_amd64.deb
```

…gives the output:

```
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Note, selecting 'keybase' instead of './keybase_amd64.deb'
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 keybase : Depends: libgconf-2-4 but it is not installable
E: Unable to correct problems, you have held broken packages.
```

I'm running Ubuntu 23.10. I've run `apt update && apt upgrade`. I've also enabled universe (as was suggested somewhere) using `add-apt-repository universe`. But it seems that libgconf-2-4 is just not available in Ubuntu 23.10.

According to `control/control` in the deb file, the version I'm trying to install is: `6.2.3-20231016183016.06cb935ee3`

**Comment by uxtops** [comment]
same here!

**Comment by zheleznovster** [comment]
When I upgrade from ubuntu 23.04 to 23.10, my copy of libgconf-2-4 stays on my system and keybase continues to work. However, when I install keybase on a fresh copy of 23.10, I encounter the issue above

**Comment by josteinaj** [comment]
It works! Thanks :+1:

**Comment by vladbairash** [comment]
works

**Comment by uxtops** [comment]
thanks man!

**Comment by baim-kargo** [comment]
works! thanks man

**Comment by BlackChristianGeek** [comment]
> add this line to your `/etc/apt/sources.list`
> 
> `deb http://cz.archive.ubuntu.com/ubuntu lunar main universe`
> 
> then `sudo apt update`
> 
> and then re-run
> 
> `sudo apt install ./keybase_amd64.deb`
> 
> keybase now should work.
> 
> then you can probably safely remove the line you added to your sources.list and then sudo apt update
> 
> (nevermind the update warnings, keybase still works afterwards)

I have no idea how you figured this out, yet I'm grateful because it worked for me, too. Thanks, and God bless,

**Comment by FatefulNur** [comment]
This man is really genius

**Comment by rsyring** [comment]
IMO, its easier to do this manually:

```
wget -q http://mirrors.kernel.org/ubuntu/pool/universe/g/gconf/libgconf-2-4_3.2.6-8ubuntu1_amd64.deb
wget -q http://mirrors.kernel.org/ubuntu/pool/universe/g/gconf/gconf2-common_3.2.6-8ubuntu1_all.deb
sudo apt install 
sudo apt install ./gconf2-common_3.2.6-8ubuntu1_all.deb ./libgconf-2-4_3.2.6-8ubuntu1_amd64.deb
```

Now you can install Keybase per their install instructions.

**Comment by compuguy** [comment]
Yeah this issue has gotten a bit more complicated. Keybase now depends on *two more* deprecated packages on top of `libgconf` and `gconf2-common`:
```
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 keybase : Depends: libappindicator1 but it is not installable or
                    libayatana-appindicator1 but it is not installable
E: Unable to correct problems, you have held broken packages.
```

**Comment by compuguy** [comment]
Hoping either #26149 or #26525 fixes some of these issues so its can be used on Ubuntu 24.10......

**Comment by compuguy** [comment]
> > add this line to your `/etc/apt/sources.list`
> > 
> > `deb http://cz.archive.ubuntu.com/ubuntu lunar main universe`
> > 
> > then `sudo apt update`
> > 
> > and then re-run
> > 
> > `sudo apt install ./keybase_amd64.deb`
> > 
> > keybase now should work.
> > 
> > then you can probably safely remove the line you added to your sources.list and then sudo apt update
> > 
> > (nevermind the update warnings, keybase still works afterwards)
> 
> That's my boy! Work like a charm!

Then you immediately have to remove/comment out that repo line, otherwise beware of dragons...and apt package conflicts...

**Comment by WilhelmFreiheitsberg** [comment]
> > > add this line to your `/etc/apt/sources.list`
> > > `deb http://cz.archive.ubuntu.com/ubuntu lunar main universe`
> > > then `sudo apt update`
> > > and then re-run
> > > `sudo apt install ./keybase_amd64.deb`
> > > keybase now should work.
> > > then you can probably safely remove the line you added to your sources.list and then sudo apt update
> > > (nevermind the update warnings, keybase still works afterwards)
> > 
> > 
> > That's my boy! Work like a charm!
> 
> Then you immediately have to remove/comment out that repo line, otherwise beware of dragons...and apt package conflicts...

Ohh that's what I did immediately, but thank you for remind it!

I am aware that it's a workaround till Keybase fix it.

**Comment by RyanZim** [comment]
Fixed in https://github.com/keybase/client/pull/26971 and released in v6.3.1; confirmed working on Ubuntu 23.10.
